### DISCUSSION TITLE
Associer:  Applications <-> Serveurs logiques DHCP/DNS ?

### DISCUSSION THREAD

**Discussion by lynch4fr** [dbarzin/mercator#960] [Ideas]
Bonjour,
Je me demandais s'il y avait une raison pour ne pas pouvoir associer à une application un objet de type "Serveurs DHCP ou DNS" (de la Vue infra. logiques) Tout comme il est déjà possible d'associer une liste de "Serveurs logiques" ? Pourrait on l'ajouter ?

Sinon je sais que l'interface de l'outil Exploration va évoluer très prochainement. Est ce qu'il sera possible de personnaliser le trait d'un flux (couleur, épaisseur, type de trait) ? Ça serait pour améliorer la lisibilité.

Un tres grand merci pour tes reponses.

**Comment by dbarzin** [comment]
Je recommande d'utiliser les objets "serveur logique" pour modéliser les serveurs DNS et DHCP, éventuellement utiliser le champ "type" pour bien les identifier. Les objets de type "Serveurs DHCP ou DNS" sont dépréciés, il est préférable de les masquer au niveau de la gestion des rôles. 

Concernant l'outil d'exploration, je prépare pour l'année prochaine un outil de dessin de la cartographie qui sera beaucoup plus complet.

**Reply by lynch4fr** [reply]
Merci, je vais suivre cette recommandation.
Bonne journée
